client/asset/interface.go

```diff
@@ -233,6 +268,28 @@ type Wallet interface {
 	RegFeeConfirmations(ctx context.Context, coinID dex.Bytes) (confs uint32, err error)
 }
 
+// Rescanner is a wallet implementation with rescan functionality.
+type Rescanner interface {
+	Rescan(ctx context.Context) error
+}
+
+// Sender is a wallet that can send funds to an address, as opposed to
+// withdrawing a certain amount from the source wallet/account.
+type Sender interface {
+	Send(address string, value, feeSuggestion uint64) (Coin, error)
+}
+
+// Sweeper is a wallet that can clear the entire balance of the wallet/account
+// to an address. Similar to Withdraw, but no input value is required.
+type Sweeper interface {
+	Sweep(address string, feeSuggestion uint64) (Coin, error)
+}
```

Comment:
Will toast these unused interfaces.  This was just feeling out the uses.

Reply:
Meh. The docs serve some purpose, even if not implemented. Nuke if you want, but I don't really mind.